src/battle_util.c

```diff
@@ -10839,10 +10840,13 @@ u8 GetBattleMoveCategory(u32 moveId)
 
     if (IS_MOVE_STATUS(moveId))
         return DAMAGE_CATEGORY_STATUS;
-    else if (gMovesInfo[moveId].type < TYPE_MYSTERY)
-        return DAMAGE_CATEGORY_PHYSICAL;
+    else if (gMain.inBattle)
+    {
+        GET_MOVE_TYPE(moveId, moveType);
+    }
```

Comment:
```suggestion
        GET_MOVE_TYPE(moveId, moveType);
```
To stay consistent across an if-else block we either use brackets everywhere or none at all.

Reply:
In that case there has to be brackets everywhere, because `GET_MOVE_TYPE` needs brackets around it.

Reply:
you can wrap `GET_MOVE_TYPE` around a while loop which resolves the problem. It generally should be done regardless of this

Reply:
honestly, get move type should probably be just a function.

Reply:
but this is out of scope for this pr